centreon/www/class/centreonAuth.LDAP.class.php

```diff
@@ -225,33 +225,35 @@ public function updateUserDn()
 
             //getting user's email
             $userEmail = $this->contactInfos['contact_email'];
-            if (
-                isset($userInfos[$this->ldap->getAttrName('user', 'email')])
-                && trim($userInfos[$this->ldap->getAttrName('user', 'email')]) != ''
-            ) {
-                if (is_array($userInfos[$this->ldap->getAttrName('user', 'email')])) {
-                    // Get the first if there are multiple entries
-                    if ($userInfos[$this->ldap->getAttrName('user', 'email')][0]) {
-                        $userEmail = $userInfos[$this->ldap->getAttrName('user', 'email')][0];
-                    }
-                } elseif ($userInfos[$this->ldap->getAttrName('user', 'email')]) {
-                    $userEmail = $userInfos[$this->ldap->getAttrName('user', 'email')];
-                }
+            $ldapEmailValue = $userInfos[$this->ldap->getAttrName('user', 'email')];
+            if (isset($ldapEmailValue)) {
+                $userEmail = (trim(is_array($ldapEmailValue) ? current($ldapEmailValue) : $ldapEmailValue));
```

Comment:
There's no guarantee that the table isn't empty
```suggestion
                $userEmail = (trim(is_array($ldapEmailValue) && $ldapEmailValue !== [] ? current($ldapEmailValue) : $ldapEmailValue));
```